scripts/exregional_run_analysis.sh

```diff
@@ -962,7 +964,7 @@ if [ ${BKTYPE} -eq 0 ] && [ "${DO_SOIL_ADJUST}" = "TRUE" ]; then  # warm start
     for ii in ${list_iolayout}
     do
       iii=`printf %4.4i $ii`
-      ln_vrfy  -snf ${fixgriddir}/fv3_grid_spec.${iii}        fv3_grid_spec.${iii}
+      ln_vrfy  -snf ${gridspec_dir}/fv3_grid_spec.${iii}    fv3_grid_spec.${iii}
```

Comment:
To make it complete, we may also want to copy fv3_grid_spec from NWGES when IO_LAYOUT_Y=1

Reply:
We do not need to do so. The grid spec file for single domain is in gsi fix directory and will not be changed. We need the subdomain grid spec files in nwges because they will change based on the IO_LAYOUT and LAYOUT setups.